### PR TITLE
Bugfix issue77

### PR DESCRIPTION
**Pull request by thennant**
Previous images showed doctors and patients tags. These tags are added manually but when shown as an image would confuse the reader to think that it is added automatically

Let's:
* remove the tags in the image to make it clearer.
* add a addDoctor image as well

### CODE REVIEW

**Comment by mandykqh** [comment]
> Previous images showed doctors and patients tags. These tags are added manually but when shown as an image would confuse the reader to think that it is added automatically

But shouldn't the tags be considered sample data? Just like how the loaded persons are sample data, the tags are also there to show readers that such tags exist.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ _With MediConnect, managing your healthcare practice has never been easier._
 
 4. Double-click the file to start the app.<br>
    A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+   ![Ui](images/Ui_updated.png)
```
Comment (StefanosAv) [inline docs/UserGuide.md:31]: The name should be "Ui.png" because the same file is also used in the main README.md

File: docs/UserGuide.md [hunk 2]
```diff
@@ -101,7 +101,9 @@ Examples:
 * `addPatient n/Ben Smith p/98353535 e/ben@abc.com ic/S1234567A a/1 Ben Street, block 13, #01-01` adds the patient `Ben Smith` to the list.
 * `addDoctor n/Sarah Tan p/99123456 e/sarah@abc.com ic/T7654321P a/Sarah Rd t/Pediatrician` adds the doctor `Sarah Tan` to the list.
 
-![AddPatient](images/AddPatient.png)
+![AddPatient](images/AddPatient_updated.png)
+
+![AddDoctor](images/AddDoctor.png)
```
Comment (StefanosAv) [inline docs/UserGuide.md:106]: I believe we should add a few tags such as "Pediatrician, Cardiologist, etc." for doctors to showcase more features in our UG.

**Comment by StefanosAv** [comment]
LGTM

**Pull request merged**